nml/generic.py

```diff
@@ -16,6 +16,24 @@
 # -*- coding: utf-8 -*-
 import sys, os, time
 
+# Enable VT100 sequences on windows consoles
+while os.name == "nt":
```

Comment:
No, I think this is worse, makes things much less clear. Is a few extra levels of indentation the worst thing in the world?